index.js

```diff
@@ -45,7 +45,7 @@ const doc = [
   '                    names. This can be used to fix some oddities',
   '                    with paths which appear in the source map',
   '                    generation process.  Accepts regular expressions.',
-  '  --with=AFTER  See --replace.',
```

Comment:
I usually set the `trailingComma` option to keep diffs simpler. Thoughts?

Reply:
I don't mind. I'll allow `comma-dangle`

Reply:
Done